### PR TITLE
Fix: removed deprecated MP4V-ES support

### PR DESCRIPTION
**Pull request by Bemly**
### Description

Update the content, removed deprecated MP4V-ES support.

#### mp4v always deprecated

i convert the MP4V-ES video encode file for 3GP container using ffmpeg according to the documentation.
```
> ffmpeg -i input.mp4 -c:v mpeg4 -vtag mp4v-es -an -f 3gp output.3gp | ffprobe output.3gp
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'output.3gp':
  Metadata:
    major_brand     : 3gp4
    minor_version   : 512
    compatible_brands: 3gp4isomiso2
  Duration: 00:03:36.50, start: 0.000000, bitrate: 316 kb/s
  Stream #0:0[0x1](und): Video: mpeg4 (Simple Profile) (mp4v / 0x7634706D), yuv420p, 640x360 [SAR 1:1 DAR 16:9], 315 kb/s, 30 fps, 30 tbr, 15360 tbn (default)
      Metadata:
        handler_name    : ISO Media file produced by Google Inc. Created on: 01/22/2024.
        vendor_id       : [0][0][0][0]
```
but, when I test this file on webpage, Firefox can't recognize the MIME format or report an error
```
Error Code: NS_ERROR_DOM_MEDIA_DEMUXER_ERR (0x806e000c)
Details: virtual RefPtr<MP4Demuxer::InitPromise> mozilla::MP4Demuxer::Init(): No MP4 audio () or video () tracks
```

图片

i have used a variety of containers one after another,

its still cant decoder video file on this webpage.

then i use HTMLMediaElement.canPlayType and MediaSource.isTypeSupported method test, 

fenix(mobile) and firefox desktop not support.

#### Theora default deprecated version

 Theora encoder will be officially deprecated on Firefox 130 Release version, 

and have been deprecated on Firefox 130.0b1 Beta version.

图片

about:support > media > Codec Support Information > Codec Name

< 129: exist Theora, about:config > media.theora.enabled default TRUE.

= 130: not exist Theora, not exist media.theora.enabled, and decode theora format video failed.

### Additional details

1.MP4 part 2 always unsupported:

https://bugzilla.mozilla.org/show_bug.cgi?id=1427278

2.MediaSource.isTypeSupported Checker website:

https://cconcolato.github.io/media-mime-support

3. Theora have been deprecated on Firefox version download:

https://ftp.mozilla.org/pub/fenix/releases/130.0b1/

### CODE REVIEW

File: files/en-us/web/media/formats/video_codecs/index.md [hunk 1]
```diff
@@ -1587,7 +1582,7 @@ The [Theora Cookbook](https://en.flossmanuals.net/ogg-theora/_full/) offers addi
               <th scope="row">Theora support</th>
               <td>3 to 121</td>
               <td>12 to 121</td>
-              <td>3.5 to 126</td>
+              <td>3.5 to 130</td>
```
Comment (hamishwillee) [inline files/en-us/web/media/formats/video_codecs/index.md:1585]: This is incorrect. It "effectively" disappeared in 126 in https://github.com/mdn/content/pull/35275 (i.e. move behind flag that was off by default".

Comment (Bemly) [reply files/en-us/web/media/formats/video_codecs/index.md]: Thank you for your correction, I'm going to close this PR.

**Comment by hamishwillee** [comment]
I'm asking about support for the MP4v format in https://bugzilla.mozilla.org/show_bug.cgi?id=1427278#c5 - it seems likely that you're right, but want to be sure.

**Comment by hamishwillee** [comment]
@Bemly Thanks, but I'm reopening until I can confirm the part around MP4V

**Comment by hamishwillee** [comment]
I'm closing again thanks.  No response from engineering in linked issue. If they do response we can reopen.

**Pull request closed without merging**